src/diffusers/pipelines/text_to_video_synthesis/pipeline_text_to_video_synth.py

```diff
@@ -0,0 +1,638 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
Do we want to attribute the ModelScope authors as well (for the pre-trained parameters)?